lib/queueClient.ts

```diff
@@ -309,4 +331,44 @@ export class QueueClient extends Client {
   async cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void> {
     return this._context.managementClient!.cancelScheduledMessages(sequenceNumbers);
   }
+
+  /**
+   * Lists the sessions on the ServiceBus Queue.
+   * @param skip The number of sessions to skip
+   * @param top Maximum numer of sessions.
+   * @param lastUpdateTime Filter to include only sessions updated after a given time. Default
+   * value: 3 days ago from the current time.
+   */
+  async listMessageSessions(skip: number, top: number, lastUpdatedTime?: Date): Promise<ListSessionsResponse> {
+    return this._context.managementClient!.listMessageSessions(skip, top, lastUpdatedTime);
+  }
+
+  /**
+   * 
+   * @param options 
+   */
+  async acceptSession(options?: AcceptSessionOptions): Promise<MessageSession> {
```

Comment:
From the code I see that this creates a receiver for given sessionId/receiver name.
But the name `acceptSession` doesn't convey the same. Is this what other SDKs call it as well?

Reply:
> Or does this use the same argument as the QueueClient vs MessageReceiver disucssion? i.e .Net meant QueueClient for simple users and MessageSender/MessageReceiver/SessionClient for power users?

Ok, just heard back from Neeraj, that this is indeed the case